project/zona8815/README.rst

```diff
@@ -0,0 +1,139 @@
+Hardware Aging Acceleration
+------------------------------------
+
+Accelerate the hardware aging simulation of the CNN MAC operations. 
+
+Basic Information
+-----------------
+
+In the research of the AI Accelerator, we sometimes need to focus on the 
+different results of the CNN MAC operations which caused by the hardware 
+aging to judge the reliability. With the aging of the MOS, the each operation 
+result of the MAC operations will become different, result in that we need 
+to build the truthtable for all inputs. 
+
+The whole simulation time will be very long since we need to find the output 
+from the truthtable, even though we just simulate a simple 2D convolution. 
+Therefore, we want to focus on how to speed up the time for the simulation. 
+
+Here are some information for this work: 
+
+- GitHub Repository: `Hardware Aging Acceleration <https://github.com/zona8815/Hardware-Aging-Acceleration>`__
+- Coding
+    - C++ and Python
+- Aging truthtable
+    - MOSRA
+
+Problem to Solve
+----------------
+
+As we want to speed up the simulation time, we can list some problems: 
```

Comment:
You talk about speedup here, which is good.  But do you have an existing open-source code as the base of the speedup?  If not, your proposal should include creating the baseline.

Reply:
It is still not clear what is the baseline of the speedup in the updated proposal.  I'd like to leave it here because it's time to start coding work.  Please keep in mind that when talking about speedup, you need a baseline defined.  It should be appearing in any speedup work.